trailblazer/clients/job_info_response.py

```diff
@@ -0,0 +1,147 @@
+from pydantic import BaseModel
+
+from trailblazer.clients.common import SlurmError, SlurmWarning
+
+
+class SlurmJobInfo(BaseModel):
```

Comment:
just wondering! - can this be fetched for a case? Would one need to gather all job ids for jobs in a case and parse this?

Reply:
I don't think slurm has any concept of a case, but yes - that functionality will likely be needed somewhere

Reply:
Great, that is information is precisely thing that we want to track with arnold for anyu given case to evaluate run time performance! Cool stuff!!

Reply:
Currently I think the job id:s are written to a file on Hasta for the case. That path is stored for the analysis in trailblazer. It seems a little messy, so would be nice to change but thats how it works currently!

Maybe this client should be a separate package, but we can change that later 👍